styles/green.css

```diff
@@ -0,0 +1,72 @@
+body {
+  display: grid;
+  grid-auto-rows: repeat(5 1fr);
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  flex-wrap: row wrap;
+  justify-content: space-between;
+}
+
+nav ul > li {
+  flex: 1;
+}
+
+nav ul li img, nav ul li {
```

Comment:
I've seen this exact CSS rule in a **lot** of files.  Try having **one** css file and sharing the styles.  It will be easier to maintain.